apps/gateway/.env.example

```
@@ -0,0 +1,40 @@
+
+# Upstash QStash
+QSTASH_TOKEN=
+QSTASH_URL=
+QSTASH_CURRENT_SIGNING_KEY=
+QSTASH_NEXT_SIGNING_KEY=
+
+# Service Auth
+GATEWAY_API_KEY=              # Console→Gateway authentication
+GATEWAY_WEBHOOK_SECRET=       # Gateway→Console HMAC signing
+GATEWAY_BASE_URL=https://gateway.lightfast.ai
+ENCRYPTION_KEY=               # AES-256-GCM for token encryption (32+ hex chars)
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)